packages/@aws-cdk/aws-apigatewayv2/lib/common/base/api.ts

```diff
@@ -0,0 +1,50 @@
+import * as cloudwatch from '@aws-cdk/aws-cloudwatch';
```

Comment:
(Missed this in the last round) 

Quite confusing to have base as another subfolder of common - `lib/common/base/api.ts`. Maybe move all the internal base classes into `lib/common/base.ts`?

Reply:
Yeah makes sense. Done.